asapdiscovery-data/asapdiscovery/data/aws/cli.py

```diff
@@ -0,0 +1,108 @@
+from pathlib import Path
+
+import click
```

Comment:
I know some of the other scripts use `argparse`, I think click will work best for a large scale cli (and we discussed as much) but we should try and enforce a consistent approach and a bit of a backport. Lets raise an issue to centralise?

Reply:
Yes, probably need a standalone issue laying out this vision; I sketched the vision [here](https://github.com/choderalab/asapdiscovery/issues/132#issuecomment-1446756002), but that might not be a clear enough signpost.